autoload/twitvim.vim

```diff
@@ -2127,9 +2127,6 @@ function! s:post_twitter(mesg, inreplyto)
     " line. Don't let it count towards the tweet length.
     let mesg = substitute(mesg, '\n$', '', "")
 
-    " Convert internal newlines to spaces.
-    let mesg = substitute(mesg, '\n', ' ', "g")
-
     let mesglen = s:mbstrlen(mesg)
```

Comment:
I'm thining that this should be optional for all users. And it should be false in default.